server_test.go

```diff
@@ -444,3 +444,50 @@ func TestDecodeContext(t *testing.T) {
 		testDecodeContext(t, h1, h2, h1.ID())
 	})
 }
+
+func TestAuthorization(t *testing.T) {
+	h1, h2 := makeRandomNodes()
+	defer h1.Close()
+	defer h2.Close()
+
+	authorizationFunc := func(pid peer.ID, svc string, method string) bool {
+		if method == "Multiply" {
```

Comment:
This should specify that a particular peer, i.e. `h2`, is allowed to make calls, and there should be a third peer that then attempts to make calls and those calls should fail.